CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# v1.5.1 (2021-12-23)
+
+## Security Fixes
+* Update hotdog to the latest release. Hotdog now mimics the permissions of the target JVM process ([#1884])
+
+## OS Changes
+* Updated host containers to the latest version ([#1882])
```

Comment:
should we also link to #1881?

Reply:
Yes! I missed that these were done as separate PRs. I'll update this PR.